### DISCUSSION TITLE
About Packages Update

### DISCUSSION THREAD

**Discussion by esraercann** [wolfi-dev/os#10465] [Q&A]
Hello, I wanted to ask something about Wolfi/os. Since there was a vulnerability in the 1.36.1-r4 package on the Busybox side, we updated the image we created with Wolfi, but the 1.36.1-r18 package in Chainguard's latest image is not [avaliable](https://packages.wolfi.dev/). How we can solve it or has there been a change on the Chainguard side?

**Comment by amouat** [comment]
The current package seems to be busybox-1.36.1-r7. Note that "r" numbers are internal to wolfi, they indicate changes we've made to the build process but that we are still using the 1.36.1 sources.
